.github/actions/after-job/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Post-job actions'
```

Comment:
So the name is "Post-job actions" -- shouldn't it live in `post-job` rather than `after-job`?